### DISCUSSION TITLE
is association of email with logged in wallet possible??

### DISCUSSION THREAD

**Discussion by devsalman247** [Web3Auth/Web3Auth#1139] [Help]
Can we associate email with wallet whom with user has logged in (User is connected with external wallet)??
will i be able to send crypto to user connected with external wallet using his associated email..If it is possible then how??

**Comment by shahbaz17** [comment]
Hey @devsalman247 

Associating email to an external wallet is not supported by us. Since logging in with socials will get you a new address. But the external wallets will already have a blockchain address. 

Have you tried our demo app? https://demo-app.web3auth.io

When you log in using any social, you will get different wallet addresses, though there is a way to get the same address for some social combinations. But email and external wallet are not possible at the web3auth level.

**Reply by devsalman247** [reply]
Thanks @shahbaz17 for help!
